clarity-ext-scripts/clarity_ext_scripts/covid/ct_analyes_service.py

```diff
@@ -0,0 +1,37 @@
+from clarity_ext.extensions import GeneralExtension
+from clarity_ext_scripts.covid.ct_discriminator import CtDiscriminator
+from clarity_ext.domain.validation import UsageError
+
+
+class CtAnalyseService(object):
```

Comment:
Maybe `QPCRAnalysisService` is a better name here, since we might actually need to take more than the Ct values into account. See my extract from the manual below.

Reply:
Right, I think it might be easier to understand from a dev's point of view too.

Reply:
ok

Reply:
I'll use RTPCR if you don't mind